taurus/demo/tests/test_sac.py

```diff
@@ -1,23 +1,23 @@
 """Test Strehlow & Cook demo."""
-from taurus.demo.strehlow_and_cook import make_strehlow_table, make_strehlow_objects
+from taurus.demo.strehlow_and_cook import make_strehlow_table, make_strehlow_objects, \
+    minimal_subset, import_table
 import taurus.client.json_encoder as je
 
 
 def test_sac():
     """Make S&C table and assert that it can be serialized."""
-    sac = make_strehlow_objects()
+    sac = make_strehlow_objects(import_table())
     sac_tbl = make_strehlow_table(sac)
 
+    # Check that all shapes of records serialize and deserialize
+    for comp in sac:
+        assert je.loads(je.dumps(comp)) == comp
+
     # Look at each different combination of Value types in a S&C record
-    seen = set()
-    for comp, row in zip(sac, sac_tbl['content']):
-        mask = ''.join(map(lambda x: str(type(x)), row))
-        if mask not in seen:
-            seen.add(mask)
-            # Check that all shapes of records serialize and deserialize
-            assert je.loads(je.dumps(comp)) == comp
+    smaller = minimal_subset(sac_tbl['content'])
     # Make sure that the diversity of value types isn't lost, e.g. something is being None'd
-    assert len(seen) == 12
+    assert len(smaller) == 153
```

Comment:
Is there a way to do this check without hardcoding 153? For example, can you count how many unique fingerprints are in the main table and use that number?

Reply:
See response below

Reply:
I do not see a response below. Did you forget to submit the comment?

Reply:
Not so much forget, as didn't get around to.  See comment below.